src/effects/LUT3DEffect.js

```diff
@@ -29,8 +29,6 @@ import fragmentShader from "./glsl/lut-3d.frag";
  * https://www.nvidia.com/content/GTC/posters/2010/V01-Real-Time-Color-Space-Conversion-for-High-Resolution-Video.pdf
  * https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/master/src/OpenColorIO/ops/lut3d/
  * https://github.com/gkjohnson/threejs-sandbox/tree/master/3d-lut
- *
- * TODO Replace DataTexture3D with Data3DTexture and rename inputEncoding to inputColorSpace.
```

Comment:
Please revert the changes regarding `inputEncoding` (that's a planned breaking change for v7).